source/frontend/src/components/maps/leaflet/Control/Search/PropertyQuickInfoContainer.tsx

```diff
@@ -0,0 +1,154 @@
+import { geoJSON } from 'leaflet';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { FaEye, FaSearchPlus } from 'react-icons/fa';
+import styled from 'styled-components';
+
+import { useMapStateMachine } from '@/components/common/mapFSM/MapStateMachineContext';
+import { SectionField } from '@/components/common/Section/SectionField';
+import TooltipWrapper from '@/components/common/TooltipWrapper';
+import usePathGenerator from '@/features/mapSideBar/shared/sidebarPathGenerator';
+import { useLtsa } from '@/hooks/useLtsa';
+import { exists, firstOrNull, isValidString, pidFormatter } from '@/utils';
+import { formatNames } from '@/utils/personUtils';
+
+export const PropertyQuickInfoContainer: React.FC<React.PropsWithChildren> = () => {
+  const [ownerNames, setOwnerNames] = useState('');
```

Comment:
Arguable here, but I prefer to store objects in their "raw" unformatted state instead of storing formatted objects intended for display - that way multiple consumers can access the raw data and format as needed. There may not be a requirement for multiple consumers here however.